product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/known_issues_notes.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: "Known issues and additional notes"
+
+---
+
+This page lists known issues and additional notes for:
+
+ - [Migration Portal](#migration-portal)  
+ - [EDB DDL Extractor](#edb-ddl-extractor) 
+ - [Data pump utility](#data-pump-utility) 
```

Comment:
Data pump is not a single utility.  It Data Pump is an oracle feature that provides a set of utilities.  When we reference it, we probably should say "Oracle Data Pump utilities" (note capitalization of "Data" and "Pump").

Reply:
Done.